bump-version/action.yml

```diff
@@ -0,0 +1,46 @@
+name: Bump Version
+description: Bump the version and push if not a dry-run
+inputs:
+  version:
+    description: The new version
+    required: true
+  version_bump_script:
+    description: The version bump script
+    required: true
+  commit_template:
+    description: The template for the git commit message
+    default: "BUMP ${VERSION}"
+  push_commit:
+    description: Whether to push the commit
+    required: true
+  artifactory_image:
+    description: "Image to use for artifactory"
+    default: release-tools-container-registry-local/garasign-git
+
+runs:
+  using: composite
+  steps:
+    - name: Set new version
+      shell: bash -eux {0}
+      run: |
+        ${{ inputs.version_bump_script }} ${{ inputs.version }}
+    - name: Get the commit message
+      shell: bash -eux {0}
+      run: |
+        export VERSION=${{inputs.version}}
+        export COMMIT_MESSAGE=$(echo "${{inputs.commit_template}}" | envsubst)
+        echo "COMMIT_MESSAGE=$COMMIT_MESSAGE" >> $GITHUB_ENV
+    - name: Commit the version bump
+      uses: mongodb-labs/drivers-github-tools/git-sign@v2
+      with:
+        command: git commit -a -m \"${{env.COMMIT_MESSAGE}}\" -s --gpg-sign=${{ env.GPG_KEY_ID }}
+        artifactory_image: ${{inputs.artifactory_image}}
+    - name: Push the commit to the source branch
+      shell: bash -eux {0}
+      run: |
+        if [ ${{ inputs.push_commit }} != "true" ]; then
```

Comment:
The logic is backwards, resulting in a push when `push_commit` is set to `false` ;)

Reply:
Updated